charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -0,0 +1,123 @@
+{{- if .Values.repoServer.enabled }}
```

Comment:
This should not be configurable, you can't disable repo-server

Reply:
This item is fixed on commit https://github.com/argoproj/argo-helm/pull/129/commits/41df9ed6725dd7ece14944bc8daff335dd365b15